endpoints/openrtb2/auction.go

```diff
@@ -1922,6 +1927,31 @@ func validateDevice(device *openrtb2.Device) error {
 	return nil
 }
 
+func validateOrFillCDep(httpReq *http.Request, req *openrtb_ext.RequestWrapper, account *config.Account) error {
+	if account == nil || !account.Privacy.PrivacySandbox.CookieDeprecation.Enabled {
+		return nil
+	}
+
+	secCookieDeprecation := httpReq.Header.Get(secCookieDeprecation)
+	if secCookieDeprecation == "" {
+		return nil
+	}
+	if len(secCookieDeprecation) > 100 {
+		return &errortypes.Warning{
+			Message:     "request.device.ext.cdep must be less than 100 characters",
```

Comment:
Message should be updated to reflect that values over 100 characters are invalid. Current text implies a cdep of exactly 100 characters is invalid, but that's allowed.

Reply:
Do you mean the message should be 
```
The value for 'request.device.ext.cdep' cannot exceed 100 characters
```

Please suggest

Reply:
I'm good with "request.device.ext.cdep must not exceed 100 characters"

Reply:
done